sys-boot/tboot/files/tboot-1.10.5-fix-pconf-element.patch

```diff
@@ -0,0 +1,21 @@
+# HG changeset patch
```

Comment:
No link to upstream submission? Can you add it in a follow-up?

Reply:
Upstream doesn't have a pull requestor or a bug reporter. Everything is done through e-mail. I can link to the e-mail in the archive though: https://sourceforge.net/p/tboot/mailman/message/37616841/